runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/SmithingBonus.java

```diff
@@ -0,0 +1,15 @@
+package net.runelite.client.plugins.skillcalculator.skills;
```

Comment:
Needs copyright header. (see existing classes for reference)

Reply:
Do I use my own name & email for this part?

Reply:
Yes, ideally it should be similar to your commit's name/email